crates/release_plz_core/src/gitea_client.rs

```diff
@@ -13,6 +13,12 @@ pub struct GiteaClient<'a> {
     client: reqwest::Client,
 }
 
+#[derive(Serialize)]
+pub struct CreateReleaseOption<'a> {
+    tag_name: String,
+    release_body: &'a str,
```

Comment:
```suggestion
    release_body: &'a str,
```
from the [API](https://try.gitea.io/api/swagger#/repository/repoCreateRelease) I see this fiels is called `body`, not `release_body`.
Was this a typo or maybe there's a reason behind this name? 🤔 
![image](https://user-images.githubusercontent.com/11428655/211922370-a4c0a0ae-0750-4436-bced-1323f95acd84.png)